asustor_it87.c

```diff
@@ -3074,6 +3355,33 @@ static const struct attribute_group it87_group_auto_pwm = {
 	.is_visible = it87_auto_pwm_is_visible,
 };
 
+static struct attribute *it87_attributes_gp_led_blink[] = {
+    &sensor_dev_attr_gpled1_blink.dev_attr.attr,
+    &sensor_dev_attr_gpled2_blink.dev_attr.attr,
+    &sensor_dev_attr_gpled1_blink_freq.dev_attr.attr,
+    &sensor_dev_attr_gpled2_blink_freq.dev_attr.attr,
+
+    NULL
+};
+
+static umode_t it87_gpled_blink_is_visible(struct kobject *kobj, struct attribute *attr, int index)
+{
+#if 0
+    // DG: no idea what this is about exactly, was already commented out in asustor code
+    // struct device *dev = container_of(kobj, struct device, kobj);
+    // struct it87_data *data = dev_get_drvdata(dev);
+    // int i = index / 7; /* temperature index */
+    // int a = index % 7; /* attribute index */
```

Comment:
I think one or the other would make sense here, either `#if 0` and uncommented, or remove `#if 0` and keep comments, up to you.

```suggestion
    struct device *dev = container_of(kobj, struct device, kobj);
    struct it87_data *data = dev_get_drvdata(dev);
    int i = index / 7; /* temperature index */
    int a = index % 7; /* attribute index */
```

Reply:
done